runbotics/runbotics-desktop/src/action/sharepoint/excel/sharepoint-excel.action-handler.ts

```diff
@@ -41,22 +41,26 @@ export default class SharepointExcelActionHandler extends StatelessActionHandler
             throw new Error(SharePointExcelErrorMessage.getCellIncorrectInput());
         }
 
-        const { value, text, numberFormat } = await this.excelService.getCell({
+        const cellValue = await this.excelService.getCell({
             column: column.toString(),
             row: row.toString()
         });
 
-        if (numberFormat.includes('@') || numberFormat.includes('%')) {
-            return text;
-        }
-
-        return value;
+        return cellValue;
     }
 
     async getRange(
         input: SharepointTypes.SharepointExcelGetRangeActionInput
```

Comment:
in Excel Desktop instead of range we accept "startCell" and "endCell". imo we should stick to it